[2022-12-18T10:18:39.013+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: Can_I_change_name_to_this.load_basic_to_pg manual__2022-12-18T10:17:01.135256+00:00 [queued]>
[2022-12-18T10:18:39.038+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: Can_I_change_name_to_this.load_basic_to_pg manual__2022-12-18T10:17:01.135256+00:00 [queued]>
[2022-12-18T10:18:39.041+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-18T10:18:39.043+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-18T10:18:39.052+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-18T10:18:39.086+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): load_basic_to_pg> on 2022-12-18 10:17:01.135256+00:00
[2022-12-18T10:18:39.104+0000] {standard_task_runner.py:55} INFO - Started process 281 to run task
[2022-12-18T10:18:39.132+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Can_I_change_name_to_this', 'load_basic_to_pg', 'manual__2022-12-18T10:17:01.135256+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpye1w4l7r']
[2022-12-18T10:18:39.147+0000] {standard_task_runner.py:83} INFO - Job 146: Subtask load_basic_to_pg
[2022-12-18T10:18:39.277+0000] {task_command.py:389} INFO - Running <TaskInstance: Can_I_change_name_to_this.load_basic_to_pg manual__2022-12-18T10:17:01.135256+00:00 [running]> on host 502390ac5559
[2022-12-18T10:18:39.390+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Can_I_change_name_to_this
AIRFLOW_CTX_TASK_ID=load_basic_to_pg
AIRFLOW_CTX_EXECUTION_DATE=2022-12-18T10:17:01.135256+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-18T10:17:01.135256+00:00
[2022-12-18T10:18:39.398+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-18T10:18:39.402+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/tasks/redshift/basic_load_to_redshift.py']
[2022-12-18T10:18:39.428+0000] {subprocess.py:86} INFO - Output:
[2022-12-18T10:18:39.781+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2022-12-18T10:18:39.783+0000] {subprocess.py:93} INFO -   File "/opt/***/tasks/redshift/basic_load_to_redshift.py", line 10, in <module>
[2022-12-18T10:18:39.784+0000] {subprocess.py:93} INFO -     from logger.Logger import Log
[2022-12-18T10:18:39.787+0000] {subprocess.py:93} INFO - ModuleNotFoundError: No module named 'logger'
[2022-12-18T10:18:39.795+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2022-12-18T10:18:39.832+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2022-12-18T10:18:39.853+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=Can_I_change_name_to_this, task_id=load_basic_to_pg, execution_date=20221218T101701, start_date=20221218T101839, end_date=20221218T101839
[2022-12-18T10:18:39.892+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 146 for task load_basic_to_pg (Bash command failed. The command returned a non-zero exit code 1.; 281)
[2022-12-18T10:18:39.974+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-12-18T10:18:40.093+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
